### PR TITLE
deploy verification manifest on preprocess success

### PR DESCRIPTION
**Pull request by lsf37**
This replaces the similar bamboo job.

### CODE REVIEW

**Comment by lsf37** [comment]
The bashism test complains about `<<<` as usual, which I'm planning to ignore as usual :-)

**Review by Xaphiosis — APPROVED** [review]

**Review by mbrcknl — APPROVED** [review]
Sorry for the slow reply. I didn't quite figure out how the different actions fit together, but it is nevertheless good to have automatic manifest deployment in GitHub!

**Comment by lsf37** [comment]
> Sorry for the slow reply. I didn't quite figure out how the different actions fit together, but it is nevertheless good to have automatic manifest deployment in GitHub!

No worries. The interplay between the actions is becoming a bit too complex, but I hope we've reached the maximum level by now. The next few deployment options should be mostly re-using things that are already there.

File: l4v-deploy/common.py [hunk 1]
```diff
@@ -40,3 +40,4 @@ def format_commit_message(msg):
 def set_repo_email(repo='.', user=git_commit_user, email=git_commit_email):
     loud_command(['git', 'config', 'user.name', user], cwd=repo)
     loud_command(['git', 'config', 'user.email', email], cwd=repo)
+    loud_command(['git', 'config', '--add', 'gerrit.createChangeId', '0'], cwd=repo)
```
[the remark this answers is not in the document]
Comment (lsf37) [reply l4v-deploy/common.py:43]: I was wondering where it came from and googled it, turns out it's just `gerrit` and no use to us at all.

**Pull request merged**